code/modules/surgery/organs/cybernetics/augments_legs.dm

```diff
@@ -0,0 +1,177 @@
+
+/obj/item/organ/cyberimp/leg/shove_resist/Remove(mob/living/carbon/M, special)
+	REMOVE_TRAIT(owner,TRAIT_SHOVE_RESIST,type)
+	return ..()
+
+/obj/item/organ/cyberimp/leg/accelerator
+	name = "P.R.Y.Z.H.O.K accelerator system"
+	desc = "Russian implant that allows you to tackle people."
+	encode_info = AUGMENT_TG_LEVEL
+	double_legged = TRUE
+	var/datum/component/tackler
+
+/obj/item/organ/cyberimp/leg/accelerator/on_full_insert(mob/living/carbon/M, special, drop_if_replaced)
+	. = ..()
+	tackler = M.AddComponent(/datum/component/tackler, stamina_cost=30, base_knockdown = 1.5, range = 5, speed = 2, skill_mod = 1.5, min_distance = 3)
+
+/obj/item/organ/cyberimp/leg/accelerator/Remove(mob/living/carbon/M, special)
+	if(tackler)
+		qdel(tackler)
+	return ..()
+
+/obj/item/organ/cyberimp/leg/chemplant
+	name = "Debug Chemplant"
+	desc = "You shouldn't see this!"
+	icon_state = "chemplant"
+	implant_overlay = "chemplant_overlay"
+	var/list/reagent_list = list()
+	var/health_threshold = 20
+	var/max_ticks_cooldown = 100
+	var/current_ticks_cooldown = 0
+
+/obj/item/organ/cyberimp/leg/chemplant/on_life()
+	if(!check_compatibility())
+		return
+
+	if(current_ticks_cooldown > 0)
+		current_ticks_cooldown--
+		return
+
+	if(owner.health < health_threshold)
+		current_ticks_cooldown = max_ticks_cooldown
+		on_effect()
+
+/obj/item/organ/cyberimp/leg/chemplant/emp_act(severity)
+	. = ..()
+	health_threshold += rand(-10,10)
+
+	on_effect()
+
+/obj/item/organ/cyberimp/leg/chemplant/proc/on_effect()
+	owner.reagents.add_reagent_list(reagent_list)
+	to_chat(owner,"<span class = 'notice'> You feel a sharp pain as the cocktail of chemicals is injected into your bloodstream!</span>")
+	return
+
+/obj/item/organ/cyberimp/leg/chemplant/drugs
+	name = "deep-vein emergency morale rejuvenator"
+	desc = "Dangerous implant used by the syndicate to reinforce their assault forces that go on suicide missions."
+	implant_color = "#74942a"
+	encode_info = AUGMENT_SYNDICATE_LEVEL
+	reagent_list = list(/datum/reagent/determination = 5, /datum/reagent/drug/methamphetamine = 5 , /datum/reagent/medicine/atropine = 5)
+
+/obj/item/organ/cyberimp/leg/chemplant/emergency
+	name = "deep emergency chemical infuser"
+	desc = "Dangerous implant used by the syndicate to reinforce their assault forces that go on suicide missions."
+	implant_color = "#2a6194"
+	encode_info = AUGMENT_NT_HIGHLEVEL
+	reagent_list = list(/datum/reagent/medicine/atropine = 5, /datum/reagent/medicine/omnizine = 3 , /datum/reagent/medicine/leporazine = 3, /datum/reagent/medicine/c2/aiuri = 2, /datum/reagent/medicine/c2/libital = 2)
+
+/obj/item/organ/cyberimp/leg/chemplant/rage
+	name = "R.A.G.E chemical system"
+	desc = "Extremely dangerous system that fills the user with a mix of potent drugs in dire situation."
+	implant_color = "#ce3914"
+	encode_info = AUGMENT_TG_LEVEL
+	reagent_list = list(/datum/reagent/determination = 5, /datum/reagent/drug/methamphetamine = 5 , /datum/reagent/drug/bath_salts = 5)
```

Comment:
this might just kill you with braindamage, why not have it give you the rage martial art and a red screen effect instead?

Reply:
5 meth + 5 bath salts shouldn't kill you
but will most likely leave you very heavily disabled.

Reply:
is it intended tho

Reply:
I actually agree with tlal, this mix is not only redundant due to bath salts rendering you immune to the effects meth only mitigates, but it probably will just never be worth utilizing to begin with since it will come dangerously close to leaving you with severe brain damage.

You are better off finding some other use for this. You personally added some new drugs, maybe you could use a few of those, or possibly just basic ephedrine. I also caution even adding an implant like this simply because it starts to edge in on the adrenal implant.

Reply:
imo, the meth should be replaced with penthrite (and maybe a dash of morphine).

Reply:
Adrenal implant is dead.